src/jsd/actuator.h

```diff
@@ -60,6 +60,13 @@ class Actuator : public JsdDeviceBase
   void      Reset() override;
   bool      SetOutputPosition(double position);
   bool      HasAbsoluteEncoder();
+  ActuatorStateMachineState GetActuatorStateMachineState();
+  double    GetActuatorStateCmdPosition();
+  void      SetActuatorCmdPosition(double pos);
+  void      SetActuatorCmdVelocity(double vel);
+  void      SetActuatorCmdCurrent(double cur);
```

Comment:
These API interfaces should not exist! the only supported way to interact with devices is by pushing commands to the command queue though the manager.